functions/src/index.ts

```diff
@@ -8,3 +9,5 @@ export const iCalFunction = functions.pubsub.schedule('0 0 * * *').onRun(getICal
 export const FocusTasksDueToday = functions.pubsub.schedule('0 0 * * *').onRun(focusTasksDueToday);
 
 export const RemoveOldTasks = functions.pubsub.schedule('0 0 * * *').onRun(removeOldTasks);
+
+export const MigrateTaskOwners = functions.pubsub.schedule('0 0 * * *').onRun(migrateTaskOwners);
```

Comment:
I don't think it should be a cron job. I think it's just something we should run once when no one is using it.

Reply:
oops, kind of messed up there. making it a cron job would actually cause errors and make the owner fields multidimensional arrays because the script assumes by default that owners are only strings 😕